### PR TITLE
Migrate Hero calling app to .NET Core 3.1.

### PR DESCRIPTION
**Pull request by rajasekaran2003**

### CODE REVIEW

File: samples/Group Calling Hero Sample/Web/Calling/Calling.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (sphenry) [inline samples/Group Calling Hero Sample/Web/Calling/Calling.csproj:4]: Also update README.md line 16:
[.NET Core 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)

Comment (rajasekaran2003) [reply samples/Group Calling Hero Sample/Web/Calling/Calling.csproj:4]: done

**Comment by alex-kwan** [comment]
Can we close this PR since we have updated to .netcore 3.1 in another PR?

**Comment by ProbablePrime** [comment]
@rajasekaran2003 @alex-kwan So what's the latest here are we going to close this?

**Pull request closed without merging**